backend/base/models.py

```diff
@@ -357,6 +357,10 @@ def clean(self):
         super().clean()
         if not self.timestamp:
             self.timestamp = datetime.now()
+        if self.type == "AA" or self.type == "BI" or type == "VE":
```

Comment:
I though multiple picture inside the building were allowed?

Reply:
Yeah multiple pictures are allowed but there may only be one remark for AA, BI & VE. Only one remarks for arrival, inside & leaving is allowed.

Reply:
My understanding was multiple picture inside, but I don't think this matters a lot. More pictures can always be uploaded under a normal comment.

Reply:
Yeah mulitple pictures of a single remark is still possible. It's just the remark (string) that is only allowed once.